### DISCUSSION TITLE
Call for Ideas: upcoming DemoDay 28th Oct 2021

### DISCUSSION THREAD

**Discussion by javabster** [Qiskit/.github#7] [DemoDay]
We have a few open slots for DemoDay on the 28th Oct. If you have an idea for something you'd like to see, or if you'd like to volunteer/nominate someone, let us know in the comments here!

DemoDays are a no-stress, casual get together, so whether you have a small work in progress PR or a fully fledged demo presentation, we'd love to hear about what you've been working on!

**Comment by HuangJunye** [comment]
I think can pick a project or two from QAMP spring or fall cohort. What about this project https://github.com/qiskit-advocate/qamp-fall-21/issues/24 I was really impressed by it.

**Reply by 1ucian0** [reply]
Is it already runnable?

**Reply by 1ucian0** [reply]
The transpiler debugger was presented on Oct 28th! Watch it [here](https://github.com/qiskit-community/feedback/wiki/Qiskit-DemoDays#oct-28-2021)
